plugins/modules/rpm_ostree_pkg.py

```diff
@@ -169,6 +217,22 @@ def main():
                 type='list',
                 elements='str',
             ),
+            allow_inactive=dict(
+                type='bool',
+                default=True,
+            ),
+            indepotent=dict(
+                type='bool',
+                default=True,
+            ),
+            apply_live=dict(
+                type='bool',
+                default=False,
```

Comment:
A default False for a boolean is redundant.

Reply:
No, it's not. It reduces a three-value option (true, false, none) to a two-value option (true, false).